File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        dic = {}
        for i, v in enumerate(nums):

            y = target - nums[i]
            if y in dic:
                return [dic[y], i]
            dic[v] = i
        

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.go
func removeDuplicates(nums []int) int {
	i,j:=0,1
	k:=1
	for i<len(nums) && j<len(nums){
		if nums[j]!=nums[i]{
			i++
			nums[i]=nums[j]
			k++
		}
		j++
	}
	return k
}
